Jailbreaking

juanrave opened the pull request:
El **jailbreaking** en el contexto de los Modelos de Lenguaje de Gran Escala (LLM, por sus siglas en inglés) se refiere a la práctica de manipular o engañar a estos modelos para que realicen acciones o generen respuestas que están fuera de sus limitaciones programadas o de sus intenciones originales.

alvarotech reviewed (approved):
Perfecto Carlos, muchas gracias! Solo falta el link en el indice general para acceder al contenido, ya lo hago yo

integralv6 committed:
Jailbreaking

El **jailbreaking** en el contexto de los Modelos de Lenguaje de Gran Escala (LLM, por sus siglas en inglés) se refiere a la práctica de manipular o engañar a estos modelos para que realicen acciones o generen respuestas que están fuera de sus limitaciones programadas o de sus intenciones originales.

dborda commented:
Hola Juan! gracias por el contenido, muy importante!

dborda committed:
Merge branch 'main' into main
